pkg/tenant/api/v1/client.go

```diff
@@ -288,6 +289,105 @@ func (s *APIv1) MemberCreate(ctx context.Context, in *Member) (out *Member, err
 	return out, nil
 }
 
+func (s *APIv1) TenantProjectList(ctx context.Context, id string, size uint32, tok string, in PageQuery) (out *TenantProjectPage, err error) {
+	if id == "" {
+		return nil, ErrMemberIDRequired
+	}
+
+	path := fmt.Sprintf("v1/tenant/%s/projects", id)
+
+	// Set values for PageSize and NextPageToken keys in PageQuery
+	in = PageQuery{size, tok}
+
+	var params url.Values
+	if params, err = query.Values(in); err != nil {
+		return nil, fmt.Errorf("could not encode query params: %w", err)
+	}
+
+	// Make the HTTP request
+	var req *http.Request
+	if req, err = s.NewRequest(ctx, http.MethodGet, path, nil, &params); err != nil {
+		return nil, err
+	}
+
+	out = &TenantProjectPage{}
+	if _, err = s.Do(req, out, true); err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
+func (s *APIv1) TenantProjectCreate(ctx context.Context, id string, in *Project) (out *Project, err error) {
+	if id == "" {
+		return nil, ErrMemberIDRequired
```

Comment:
Same as above, `ErrProjectIDRequired`?

Reply:
This will also be ErrTenantIDRequired and then ErrProjectIDRequired will be used for the remaining definitions.